### ISSUE TITLE
Trouble submitting comments on article

### ISSUE THREAD

**Issue by jessicaivins** [alistapart/AListApart#427]
Hi folks! I'm having trouble submitting a comment on this page: https://alistapart.com/article/conversations-with-robots

I logged in. Every time I try to submit the comment, the page reloads, bringing me to the top, and my comment is not there. 

I'm using Chrome on MacOS. Thanks!

**Comment by zeldman** [comment]
Hi, @jessicaivins!

Your comment was successfully added ... twice! It isn't showing up yet because of a Cloudflare caching problem.

(For the same reason, two spam "comments" I deleted earlier today are still showing up, even though they are closed in the CMS. Again, Cloudflare is to blame.)

I’ll empty the Cloudflare cache manually—our plug-in no longer works (yada yada)—and it should take care of all the problems. (Until the next nice person posts a comment and doesn't see it... grrr... )

Nice to hear from you, however adverse the circumstances...

jeffrey

cc @aarongustafson

**Comment by zeldman** [comment]
![image](https://user-images.githubusercontent.com/2983799/51200973-e1a93780-18c8-11e9-898f-3f3111900b06.png)

**Comment by stegrainer** [comment]
Just tried purging it - sometimes I'll add both the www and leave it off for the page. A force-reload afterward seemed to show it loading properly (though with both of Jessica's comments intact).

**Comment by zeldman** [comment]
Cool, Ste. Can you remove Jess’s redundant comment and force-reload again? (I'm
in a meeting for the next couple hours…)







On Tue, Jan 15, 2019 1:39 PM, Ste Grainer notifications@github.com  wrote:
Just tried purging it - sometimes I'll add both the www and leave it off for the
page. A force-reload afterward seemed to show it loading properly (though with
both of Jessica's comments intact).

—
You are receiving this because you were assigned.
Reply to this email directly, view it on GitHub, or mute the thread.



********************************
studio.zeldmanThe Yard
246 5th Ave
Suite 404
New York, NY 10001
917-746-4838An Event Apart |A List Apart |A Book Apart

**Comment by stegrainer** [comment]
Yup, done. It *seems* to be working, though there was a brief moment where one browser loaded the original page with the two now-deleted spam comments ...

**Comment by zeldman** [comment]
Thank you, Ste!


 Sent by 🦄.

**Comment by jessicaivins** [comment]
Thanks, everyone!
